src/import/FSHImporter.ts

```diff
@@ -111,6 +111,7 @@ export class FSHImporter extends FSHVisitor {
       docs.push(doc);
 
       // Create and store context for main import process
+      if (!rawFSH.content.endsWith('\n')) rawFSH.content += '\n';
```

Comment:
I'd really like to see a comment here indicating _why_ we're tacking on the newline.  A few years into the future, this line may look really suspicious to someone.

Reply:
Also, we could avoid removing the `readonly` from `RawFSH` by changing this to:
```ts
const content = rawFSH.content.endsWith('\n') ? rawFSH.content + '\n';
const ctx = this.parseDoc(content, rawFSH.path);
```
This also means we're not mutating the original input.